sqldelight-gradle/android-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightAndroidPlugin.kt

```diff
@@ -32,6 +32,20 @@ import java.io.File
 
 class SqlDelightAndroidPlugin : SqlDelightPlugin() {
   override fun apply(project: Project) {
+    if (!project.plugins.hasPlugin("android")) {
```

Comment:
Does this work if you use `com.android.application`? Also this prevents you from putting the SQL Delight plugin first.

Perhaps we should have a single Gradle plugin and then react to whichever other plugins you apply. And then in afterEvaluate if you haven't applied any we know about then we error.

Reply:
I was thinking about having a single gradle plugin as well. I think thats a good idea. Lets merge this for now and then rejog things to work as a single plugin.

Reply:
Test below makes it seem like yes.

I can work on moving to a single plugin that's reactive if you want post-merge/alpha4.

Reply:
Sounds good 👍